python/lsst/summit/utils/nightReport.py

```diff
@@ -199,8 +199,8 @@ def getExpRecordDictForDayObs(self, dayObs: int) -> dict:
         expRecords = self.butler.registry.queryDimensionRecords(
             "exposure", where="exposure.day_obs=dayObs", bind={"dayObs": dayObs}, datasets="raw"
         )
-        expRecords = list(expRecords)
-        records = {e.seq_num: e.toDict() for e in expRecords}  # not guaranteed to be in order
+        listExpRecords = list(expRecords)
+        records = {e.seq_num: e.toDict() for e in listExpRecords}  # not guaranteed to be in order
```

Comment:
Again, I kind of hate this type of workaround. Is there not a more elegant way to tell mypy we're changing types than to make new variables?

Reply:
I've tried several other things and didnt work, but I will as in #sofware-dev